app/controllers/court_applications_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CourtApplicationsController < ApplicationController
+  before_action :load_and_authorize_application
+
+  add_breadcrumb :search_filter_breadcrumb_name, :new_search_filter_path
+  add_breadcrumb :search_breadcrumb_name, :search_breadcrumb_path
+
+  def show
+    add_breadcrumb @application.application_title
+    @date_sort_direction = params.fetch(:date_sort_direction, "asc")
+  end
+
+  private
+
+  def load_and_authorize_application
+    @application = CourtDataAdaptor::Query::CourtApplication.new(params[:id]).call
+    authorize! :show, @application
+  rescue JsonApiClient::Errors::ServiceUnavailable => e
+    Sentry.capture_exception(e)
```

Comment:
Can we add some logging before the Sentry capture?

Reply:
Good idea. I've added some.